extensions/amp-story/amp-story.md

```diff
@@ -362,6 +374,46 @@ Usage: `<amp-story ... supports-landscape>...</amp-story>`
   </amp-anim>
 </figure>
 
+### Live stories
+
+If the `live-story` attribute is specified on the `<amp-story>` element, it will allow users to receive real-time updates coming from the server. It updates the client device without having to refresh the page and shows a notification when there are updates available and the user is on the last page of the story.
```

Comment:
Same comments about the wording: "server" and "real time updates" are very broad
Can we do something like the amp-live-list documentation introduction? https://amp.dev/documentation/components/amp-live-list#behavior

Reply:
Done.